WORKFLOW_USER_GUIDE.md

```
@@ -843,33 +843,29 @@ The Workflows repository includes maintenance workflows that handle sync, update
 
 ---
 
-### `maint-dependabot-auto-label.yml` - Label Dependabot PRs
-**Purpose:** Auto-labels Dependabot PRs by category
+### `maint-auto-label-dep-prs.yml` - Label dependency PRs
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)